modules/portmaster/src/app/layout/navigation/navigation.html

```diff
@@ -1,29 +1,21 @@
 <div class="logo" routerLink="monitor">
   <svg [class.connected]="(connected$ | async)" data-name="Layer 1" viewBox="0 0 128 128" class="spin reverse">
     <g data-name="Main" fill-rule="evenodd">
-      <path shape-rendering="geometricPrecision" fill="#fff" class="inner"
-        d="M176.11 36.73l-5-8.61a41.53 41.53 0 00-14.73 57.22l8.55-5.12a31.58 31.58 0 0111.19-43.49z"
-        transform="translate(-127.99 .01)" style="isolation:isolate" opacity=".8"></path>
-      <path shape-rendering="geometricPrecision" fill="#fff" class="inner"
-        d="M222.36 72.63a31.55 31.55 0 01-45 19.35l-4.62 8.84a41.54 41.54 0 0059.17-25.46z"
-        transform="translate(-127.99 .01)" style="isolation:isolate" opacity=".8"></path>
+      <path shape-rendering="geometricPrecision" fill="#fff" class="inner" d="M176.11 36.73l-5-8.61a41.53 41.53 0 00-14.73 57.22l8.55-5.12a31.58 31.58 0 0111.19-43.49z" transform="translate(-127.99 .01)" style="isolation:isolate" opacity=".8"></path>
+      <path shape-rendering="geometricPrecision" fill="#fff" class="inner" d="M222.36 72.63a31.55 31.55 0 01-45 19.35l-4.62 8.84a41.54 41.54 0 0059.17-25.46z" transform="translate(-127.99 .01)" style="isolation:isolate" opacity=".8"></path>
```

Comment:
This just seems to be a formatting change.
These are very noisy and make PRs harder to read - we need to unify our toolset asap.

Reply:
I did mention this before I started. We need to figure out how we wish to handle SVG's.